### PR TITLE
5158 styling of project groups in steps table

### PR DESCRIPTION
**Pull request by emitche**
- Style step groups and spacers in timeline table
- Add cog icon to step groups as part of Redmine issue 5000

### CODE REVIEW

File: app/assets/stylesheets/admin/loans/timeline/_table.scss [hunk 1]
```diff
@@ -1,11 +1,34 @@
...
 
-  .date-col {
-    min-width: 8rem;
+  td {
+    border: 1px solid $table-border-color;
+
+    &.spacer {
+      background: $background-gray;
+      border: none;
+    }
```
Comment (hooverlunch) [inline app/assets/stylesheets/admin/loans/timeline/_table.scss]: Nicely handled.

File: app/assets/stylesheets/admin/loans/timeline/_table.scss [hunk 2]
```diff
@@ -1,11 +1,34 @@
...
+  }
+
+  .fa-cog {
+    color: $gray3;
+    font-size: 1.2em;
```
Comment (hooverlunch) [inline app/assets/stylesheets/admin/loans/timeline/_table.scss]: Can these be a little lighter and smaller please. They have too much visual weight currently. The viewer's focus should be on the name of the group. They just need to be able to find the settings icon if they are seeking it.

File: app/views/admin/loans/timeline/table/_step.html.slim [hunk 3]
```diff
@@ -4,7 +4,7 @@
   / Add spacer for step who has a group as a sibling
   - unless siblings_are_all_steps
     td.spacer colspan="#{max_depth - entry.depth}"
-  td.project-step = entry.summary
+  td = entry.summary
```
Comment (hooverlunch) [inline app/views/admin/loans/timeline/table/_step.html.slim]: Hooray for simplifying!!!

File: app/assets/stylesheets/admin/loans/timeline/_table.scss (hunk 2 above)
Comment (emitche) [inline app/assets/stylesheets/admin/loans/timeline/_table.scss]: I went ahead and used the new lightest gray we are using and reduced the text size.

Just a note, the lighter grays, including the one in mockup of #c0c0c0/ rgb(192, 192, 192) are not accessible on a white background. See http://webaim.org/resources/contrastchecker.

**Pull request merged**